### PR TITLE
Add chain state in navbar in missions interface

### PR DESCRIPTION
**Pull request by Ishagoyal**
Added chain state in the navbar
Styled address in with accordance to the design
Changed the profile popout position when the wallet is connected and disconnected
Navbar height fix

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -128,19 +138,35 @@ const Header = () => {
 };
 
 const StatusIcon = ({ connector }) => {
-  if (connector === argentX) {
-    return <SvgIcon component={argentXIcon} />;
+  if (connector.options.id === 'argentX') {
```
Comment (vnaysngh-mudrex) [inline src/components/Header/Header.jsx:141]: its always a good practice to put checks where we have deep chaining, use optional chaining in this case

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -58,25 +58,25 @@ const Header = () => {
...
+            backgroundColor={{ xs: '#212429', md: 'transparent' }}
+            justifyContent={{ xs: 'flex-start', md: 'flex-end' }}
             px={{ xs: 2, md: 0 }}
             py={{ xs: 2, md: 0 }}
           >
             {/* <LanguageSwitcher /> */}
             {/* <HeaderWallet> */}
             <Stack direction="row" alignItems="flex-start" gap={5}>
-              {location.pathname === "/mission" ? (
+              {location.pathname === '/mission' ? (
```
[the remark this answers is not in the document]
Comment (Ishagoyal) [reply src/components/Header/Header.jsx:78]: @vnaysngh-mudrex This was already written by Illya, I did not modify it. We can create a separate task for it.

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -128,19 +138,35 @@ const Header = () => {
...
+        style={{
+          width: '16px',
+          height: '16px',
+        }}
+      />
+    );
   }
-  if (connector === braavosWallet) {
-    return <SvgIcon component={braavosIcon} />;
+  if (connector.options.id === 'braavos') {
```
[the remark this answers is not in the document]
Comment (Ishagoyal) [reply src/components/Header/Header.jsx]: @vnaysngh-mudrex already added optional chaining to it, the code is outdated.

File: src/components/Header/Header.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (vnaysngh-mudrex) [reply src/components/Header/Header.jsx:78]: yes, please create a task for this

File: src/components/ProfilePopout/ProfilePopout.jsx [hunk 4]
```diff
@@ -22,7 +24,7 @@ const ProfilePopout = () => {
   };
 
   return (
-    <ProfilePopoutContainer ref={ref}>
+    <ProfilePopoutContainer ref={ref} status={status}>
```
[the remark this answers is not in the document]
Comment (Ishagoyal) [reply src/components/ProfilePopout/ProfilePopout.jsx:27]: @vnaysngh-mudrex I am passing it as props here to use them in styled-components.

**Review by vnaysngh-mudrex — APPROVED** [review]

**Pull request merged**